src/roadmap/Roadmap.test.js

```diff
@@ -0,0 +1,32 @@
+import { render, screen } from '@testing-library/react';
+import { Roadmap } from './Roadmap';
+import '@testing-library/jest-dom/extend-expect';
+
+describe('roadmap initial state', () => {
+  test('Roadmap has title called Roadmap', () => {
+    render(<Roadmap />);
+    const roadmapTitle = screen.getByRole('heading', { name: 'Roadmap' });
+    expect(roadmapTitle).toHaveTextContent('Roadmap');
+  });
+  test.skip('Roadmap should have a button called View', () => {
```

Comment:
Did you mean to check this in as skipped?

Reply:
No, unskipped!